src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -103,6 +103,77 @@ private static DistroInfo ParseOsReleaseFile()
                     }
                 }
             }
+            else 
```

Comment:
can you also add 
```
PlatformDetection.IsRHEL
PlatformDetection.IsNotRHEL
PlatformDetection.IsRHEL9
PlatformDetection.IsNotRHEL9
```
or whatever you think we'll need?

Reply:
we'll add these when needed. do you know any place today need it? just curios.

Reply:
ok, I'll add these according to your comment.

Reply:
https://github.com/dotnet/corefx/issues/21920 perhaps ? 
https://github.com/dotnet/corefx/issues/23630
you could fix both of those if you like

for the globalization ones, I guess you're using ICU version, which is better.

Reply:
just one; https://github.com/dotnet/corefx/issues/23628

Reply:
The Globalization one I'll fix it with checking ICU version as this is more reliable. 

For the other 2 issues, do we just need to skip these tests on red hat? or we need to do something more? 

CC @JeremyKuhne for System.IO issues

Reply:
I think #23630 does as `CanGetSetPipeSz` is returning false. I don't know about #21920 - maybe RHEL doesn't support read only volumes. @ianhays